### PR TITLE
Multiple improvements to Docs

### PR DESCRIPTION
**Pull request by nastra**
This change is [Reviewable](https://reviewable.io/reviews/projectnessie/nessie/2019)

### CODE REVIEW

**Review by snazy — COMMENTED** [review]
Is this a dup of #2017? Looks very similar

**Comment by nastra** [comment]
> Is this a dup of #2017? Looks very similar

It's not a dup, it just happened to contain the same commit by accident. The second commit is what this PR is actually about. I'll rebase in a bit.

File: site/docs/tools/index.md [hunk 1]
```diff
@@ -4,16 +4,14 @@ Nessie is focused on working with the widest range of tools possible. If a tool
...
 
-- [Iceberg Integration](iceberg/index.md).
-    - [Spark](iceberg/spark.md).
-
-- [Delta Lake Integration](deltalake/index.md).
-    - [Spark](deltalake/spark.md).
-
-- [Nessie CLI](cli.md).
-- [Nessie Web UI](ui.md).        
+- [Iceberg Integration](iceberg/index.md)
```
Comment (snazy) [inline site/docs/tools/index.md:7]: Has anything changed here? GH shows a diff, but I don't see any change?

File: site/docs/tools/index.md [hunk 2]
```diff
@@ -37,4 +35,4 @@ which currently exposes both of these formats.
 
 ## Demos
 
-In [Nessie Demos](https://github.com/projectnessie/nessie-demos) github repository, we have set of demos to help the users to see how Nessie works.
+The [Nessie Demos](https://github.com/projectnessie/nessie-demos) GitHub repository contains a set of demos that help users understanding how Nessie works.
```
Comment (snazy) [inline site/docs/tools/index.md:38]:
```suggestion
The [Nessie Demos](https://github.com/projectnessie/nessie-demos) GitHub repository contains a set of demos that help users understand how Nessie works.
```

File: site/docs/try/configuration.md [hunk 3]
```diff
@@ -88,4 +88,4 @@ Metrics are published using prometheus and can be collected via standard methods
 
 ### Swagger
```
Comment (snazy) [inline site/docs/try/configuration.md:89]:
```suggestion
### Swagger UI
```

File: site/docs/tools/index.md (hunk 1 above)
Comment (nastra) [reply site/docs/tools/index.md:7]: I just removed the `.` at the end of each line

**Review by snazy — APPROVED** [review]

**Review by snazy — APPROVED** [review]

**Pull request merged**